Yelp/YelpClient.swift

```diff
@@ -84,4 +84,26 @@ class YelpClient: BDBOAuth1RequestOperationManager {
                 completion(nil, error)
         })!
     }
+    
+    func searchWithTermAndLocation(term: String, location: String, completion: ([Business]!, NSError!) -> Void) -> AFHTTPRequestOperation {
+        // For additional parameters, see http://www.yelp.com/developers/documentation/v2/search_api
+        
+        // Default the location to San Francisco
+        var parameters: [String : AnyObject] = ["term": term]
+                
+        //change ll to location (ll return nil)
+        parameters["location"] = location;
+
+        
+        print(parameters)
+        
+        return self.GET("search", parameters: parameters, success: { (operation: AFHTTPRequestOperation!, response: AnyObject!) -> Void in
+            let dictionaries = response["businesses"] as? [NSDictionary]
+            if dictionaries != nil {
+                completion(Business.businesses(array: dictionaries!), nil)
+            }
+            }, failure: { (operation: AFHTTPRequestOperation?, error: NSError!) -> Void in
+                completion(nil, error)
+        })!
+    }
 }
```

Comment:
If you're adding a convenience method, can you chain everything through the same method? That way there is less duplicate code and less extraneous code paths where bugs can crop up.

Also, can you remove any print statements that you added for testing?